java/src/main/java/com/microsoft/graph/bulk/impl/BulkGremlinObjectMapper.java

```diff
@@ -0,0 +1,29 @@
+package com.microsoft.graph.bulk.impl;
```

Comment:
It is important to add the correct license headers - since this is now an official sample. 
Can you please copy paste the header from here - https://github.com/Azure-Samples/azure-cosmos-java-sql-api-samples/blob/main/src/main/java/com/azure/cosmos/examples/common/AccountSettings.java#L1
And include it in all the java source files.